Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml

```diff
@@ -9724,6 +9726,10 @@
             <LayoutButtons Face="MovePatrol" Type="AbilCmd" AbilCmd="move,Patrol" Row="0" Column="3"/>
             <LayoutButtons Face="BurrowDown" Type="AbilCmd" AbilCmd="AP_BurrowInfestedAbominationDown,0" Row="2" Column="4"/>
             <LayoutButtons Face="AP_AberrationTowering" Type="Passive" Row="2" Column="0"/>
+            <LayoutButtons Face="AP_AberrationProtectiveCover" Type="Passive" Row="2" Column="1" Requirements="AP_HaveAberrationArmorAura"/>
+            <LayoutButtons Face="AP_AberrationIncubateBanelings" Type="Passive" Row="2" Column="2" Requirements="AP_HaveAberrationBanelingIncubation"/>
+            <LayoutButtons Face="AP_AberrationMonstrousResilience" Type="Passive" Row="2" Column="3" Requirements="AP_HaveAberrationMonstrousResilience"/>

```

Comment:
Good point! Thanks for pointing that out... now that you mention it, most passive upgrades end up on the middle row anyway, so it makes sense to put these there for consistency.

Reply:
Done; I shifted all of the upgrade icons (ie. not the Towering passive) up to the middle row. I also swapped the positions of Protective Cover and the regeneration upgrade, because Protective Cover goes away while burrowed and it would look strange to have a missing icon in the middle.